installing/installing_aws_user_infra/installing-aws-user-infra.adoc

```diff
@@ -113,6 +113,13 @@ include::modules/installation-registry-storage-non-production.adoc[leveloffset=+
 
 include::modules/installation-aws-user-infra-installation.adoc[leveloffset=+1]
 
+[id="installing-custom-dns-records"]
+== Creating custom DNS records
+
+If you must create the DNS entries for your cluster, create them now.
```

Comment:
This probably needs to go into `modules/installation-aws-user-infra-installation.adoc` so it can happen after removing bootstrap resources and before `wait-for install-complete`.

Reply:
Then why's yours after `wait-for install-complete`?

Reply:
> Then why's yours after `wait-for install-complete`?

Fix in flight openshift/installer#2496 :)